src/handlers/extract/handler.ts

```diff
@@ -1,14 +1,78 @@
-export const handler = async (event?: unknown): Promise<string | undefined> => {
-  console.log("Event:", event);
-
-  if (
-    String(process.env.EXTRACTION_HANDLER_THROW_ERROR).toLowerCase() === "true"
-  ) {
-    const message =
-      "Environment variable EXTRACTION_HANDLER_THROW_ERROR is true";
-    console.error(message);
-    throw new Error(message);
+import * as AWS from "aws-sdk";
+import { S3Event, SQSEvent } from "aws-lambda";
+import { Response } from "../../shared/types";
+
+export const handler = async (event: SQSEvent): Promise<Response> => {
+  // Set Up
+  const textExtractRole = process.env.TEXTRACT_ROLE;
+  const snsTopic = process.env.TEXTRACT_SNS_TOPIC;
+
+  if (textExtractRole === undefined || textExtractRole === "") {
+    const textractMessage = "Textract role not set.";
```

Comment:
I think you can just inline the variable in the throw. The variable name is not adding much.